docs/ExamplePatterns.md

```diff
@@ -0,0 +1,37 @@
+# Example patterns
```

Comment:
useful to say "Seal Policy Example Patterns"? not sure if its clear this is just a pattern for the move (excluding patterns on how to write a frontend using them)

Reply:
Changed